src/families/polkadot/test-dataset.js

```diff
@@ -252,25 +252,6 @@ const dataset: DatasetTest<Transaction> = {
                 warnings: {},
               },
             },
-            {
-              name: "bond extra - success but warn all funds",
```

Comment:
Ok, but now we no longer have "success" tests anymore, because of the balance being too low on bot.

@gre can we do something about it ?
This PR requires having 0.1DOT available after transactions, so all existing tests cannot meet this predicate, we have either a warning, or an error.

Reply:
Hmm... I could update the `bond extra - not enough spendable` test, with an amount that is right between "available" and "available - 0.1 DOT"?

Reply:
i don't understand because the test-dataset is supposely a static seed, not the one used by bot.
maybe we can send a bit on that sending account then. is it a static account? is it set up by @Arnaud97234  or not really? thx